spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/TaskExecutionController.java

```diff
@@ -97,7 +97,7 @@ public TaskExecutionController(TaskExplorer explorer, TaskService taskService,
 	@ResponseStatus(HttpStatus.OK)
 	public PagedResources<TaskExecutionResource> list(Pageable pageable,
 			PagedResourcesAssembler<TaskJobExecutionRel> assembler) {
-		Page<TaskExecution> taskExecutions = explorer.findAll(pageable);
+		Page<TaskExecution> taskExecutions = this.taskService.findAll(pageable);
```

Comment:
> I can move the logic into the controller, but what is wrong with the methods being in the TaskService?
The reason I asked it because of the need to duplicate those two methods (that were part of TaskExplorer) again to the TaskService. 

> Spring Cloud Task shouldn't have a TaskExplorer that is specific to a particular app, in this case a password redaction. 

But, given we are fixing this only at the SCDF controller side of things where TaskExecution details are retrieved and we always have a `TaskExplorer` I was leaning towards having a custom TaskExplorer that `extends` the `SimpleTaskExplorer` we already use in SCDF.